utils/src/server/rpc_module.rs

```diff
@@ -349,96 +350,113 @@ impl Methods {
 	/// Helper to call a method on the `RPC module` without having to spin up a server.
 	///
 	/// The params must be serializable as JSON array, see [`ToRpcParams`] for further documentation.
-	pub async fn call_with<Params: ToRpcParams>(&self, method: &str, params: Params) -> Option<String> {
-		let params = params.to_rpc_params().ok();
-		self.call(method, params).await
-	}
-
-	/// Helper alternative to `execute`, useful for writing unit tests without having to spin
-	/// a server up.
-	pub async fn call(&self, method: &str, params: Option<Box<RawValue>>) -> Option<String> {
-		let req = Request::new(method.into(), params.as_deref(), Id::Number(0));
-
-		let (tx, mut rx) = mpsc::unbounded();
-		let sink = MethodSink::new(tx);
-
-		if let MethodResult::Async(fut) = self.execute(&sink, req, 0) {
-			fut.await;
+	///
+	/// Returns the decoded value of the `result field` in JSON-RPC response if succesful.
+	///
+	/// ```
```

Comment:
Love the example addition! To keep with how Rust docs these things, how about adding an

```
# Example
```

header before it?

Reply:
(I guess this comment would apply to other examples, too)

Reply:
sure why not :)